ApiFirst/ApiFirst/RepositoryList.cs

```diff
@@ -0,0 +1,109 @@
+﻿using DomainModels.Models;
+using DomainServices.Interfaces;
+
+namespace DomainServices.Repositories
+{
+    public class RepositoryList : ICustomerRepository
+    {
+        private readonly List<Customer> List = new();
+
+        public bool Create(Customer model)
+        {
+            model.Id = GetAll().LastOrDefault()?.Id + 1 ?? 0;
+
+            bool exist = GetAll().Any(customer => customer.Cpf == model.Cpf || customer.Email == model.Email);
```

Comment:
```suggestion
            bool exist = _customers.Any(customer => customer.Cpf == model.Cpf || customer.Email == model.Email);
```
Podemos fazer essa validação em um método separado e chamá-lo aqui. Usar a lista ao invés do método GetAll()